src/configs/redisClient.js

```diff
@@ -0,0 +1,13 @@
+import redis from 'redis';
+import { config } from 'dotenv';
+
+config();
+
+const redisClient = redis.createClient({
+  url: process.env.REDIS_URL,
+  password: process.env.REDIS_PWD,
+});
```

Comment:
Destructure REDIS_URL, and REDIS_PWD from process.env

Reply:
Done, thanks